# Python code to demonstrate the exception of
# sum()
arr = ["a"]

# start parameter is not provided
Sum = sum(arr)
print(Sum)

# start = 10
Sum = sum(arr, 10)
print(Sum)
